bot/beltbot.py

```diff
@@ -61,7 +61,7 @@
 
 
 PUNCTUATION = set(punctuation)
-BELT_REQUEST_REGEX = "^(?P<belt>{belts})(?P<reason>.*)".format(belts="|".join(map(re.escape, ALL_BELTS)))
+BELT_REQUEST_REGEX = re.compile("^(?P<belt>{belts})(?P<reason>.*)".format(belts="|".join(map(re.escape, ALL_BELTS))), re.DOTALL)
```

Comment:
I really need to add a contributors file that requires a link to pythex / whatever other regex tool with working examples :D